src/components/Job/index.tsx

```diff
@@ -92,6 +94,8 @@ export default ({
                 <Grid item xs={12}>
                   <Divider variant="middle" className={classes.spaceBottom} />
                   {positionName}
+                  <br />
+                  (<TimeAgo datetime={postDate} />)
```

Comment:
I think it would be much better if we put it inside a <i> tag, what do you think?

Reply:
Made the changes ⚡

Reply:
🚀